styles.css

```diff
@@ -826,6 +864,93 @@
     justify-content: center;
   }
 
+  .popupcontainer {
+    visibility: hidden;
+    z-index: 100;
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    overflow: scroll;
+    left: 0;
+    right: 0;
+    top: 0;
+    bottom: 0;
+    background: rgba(193, 199, 208, 0.75);
+    backdrop-filter: blur(7px);
+  }
+
+  .popup {
+    display: flex;
+    flex-direction: column;
+    padding: 24px;
+    width: 1156px;
+    height: 1163px;
+    position: absolute;
+    left: 25%;
+    right: 25%;
+    top: 6%;
+    bottom: 10%;
+    background: white;
+    border: 1px solid #dfe1e6;
+    box-sizing: border-box;
+    border-radius: 16px;
+    transition-property: all;
+    transition-duration: 0.3s;
+    flex-shrink: 1;
+  }
```

Comment:
- [x] I have noticed we are struggling with adjusting the look of the popup window, I went ahead and investigated the problem for you, I could appreciate why we were struggling it was not an easy task, here's what I did and worked for me maybe try it. I noticed the problem is with our image being too large it was occurred due to we are giving a fixed height to our pop up window as you can see here  
![image](https://user-images.githubusercontent.com/87689547/152627452-8209a846-77e7-44bd-a927-fb3a2b809c81.png)
If we unset the height or used `percentage` instead this will make look neater and fix all our problems. 
I would advise you not to set a hardcoded heights they sometimes cause issues with the layout that will resolve in a hard time of figuring out the problem